### ISSUE TITLE
Apple Pie

### ISSUE THREAD

**Issue by gohkenytp** [DawnTeamMC/Promenade#114]
### What is your idea about?

What if we include Apple Pies to Minecraft? It will have the same nutrition as Pumpkin Pie except the recipe is made with apples instead of pumpkins.

**Comment by Hugman76** [comment]
Hmm I absolutely love apple pie IRL too... I eat at least ONE per week

**Comment by Hugman76** [comment]
No, because originally Promenade was meant to be centered around world gen, biomes, creature and structures. Foodstuff was meant to be exclusively included in Culinaire.
But now that I think of it, I might move Culinaire's content over to Promenade.
